From aa78e09aec4787e676ed9bf175a6cd0a892e7c28 Mon Sep 17 00:00:00 2001
From: s0dyy <maxime.sorin@clever-cloud.com>
Date: Sun, 23 May 2021 01:23:42 +0200
Subject: [PATCH] Fix arg --with-protobuf

---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 915030af4..2fba22643 100644
--- a/configure.ac
+++ b/configure.ac
@@ -956,7 +956,7 @@ dnl Detect if protobuf-c installed
 dnl ===========================================================================
 
 CHECK_PROTOBUF=yes
-HAVE_PROTOBUF=yes
+HAVE_PROTOBUF=no
 HAVE_GEOBUF=no
 
 AC_ARG_WITH([protobuf],
@@ -967,6 +967,7 @@ dnl User didn't turn off protobuf support so...
 if test "$CHECK_PROTOBUF" != "no"; then
 
 	dnl Need to find libdir, incdir and protoc-c compiler
+    HAVE_PROTOBUF=yes
 
 	CPPFLAGS_SAVE="$CPPFLAGS"
 	LDFLAGS_SAVE="$LDFLAGS"
-- 
2.31.1

