From 3a0a793ed6157f9da9b5a7610a3930cf4609851d Mon Sep 17 00:00:00 2001
From: s0dyy <maxime.sorin@clever-cloud.com>
Date: Mon, 9 Dec 2019 16:45:19 +0100
Subject: [PATCH] CORESETUP-Clang-Banned-By-Distribution

---
 src/corehost/build.sh | 22 ++--------------------
 1 file changed, 2 insertions(+), 20 deletions(-)

diff --git a/src/corehost/build.sh b/src/corehost/build.sh
index fb503e7..8ebf6e4 100755
--- a/src/corehost/build.sh
+++ b/src/corehost/build.sh
@@ -207,26 +207,8 @@ __intermediateOutputPath="$__baseIntermediateOutputPath/$__base_rid.$__configura
 export __CrossToolChainTargetRID=$__base_rid
 
 # Set up the environment to be used for building with clang.
-if command -v "clang-3.5" > /dev/null 2>&1; then
-    export CC="$(command -v clang-3.5)"
-    export CXX="$(command -v clang++-3.5)"
-elif command -v "clang-3.6" > /dev/null 2>&1; then
-    export CC="$(command -v clang-3.6)"
-    export CXX="$(command -v clang++-3.6)"
-elif command -v "clang-3.9" > /dev/null 2>&1; then
-    export CC="$(command -v clang-3.9)"
-    export CXX="$(command -v clang++-3.9)"
-elif command -v "clang-5.0" > /dev/null 2>&1; then
-    export CC="$(command -v clang-5.0)"
-    export CXX="$(command -v clang++-5.0)"
-elif command -v clang > /dev/null 2>&1; then
-    export CC="$(command -v clang)"
-    export CXX="$(command -v clang++)"
-else
-    echo "Unable to find Clang Compiler"
-    echo "Install clang-3.5 or clang3.6 or clang3.9 or clang5.0"
-    exit 1
-fi
+export CC="${CC}"
+export CXX="${CXX}"
 
 if [ ! -f $__versionSourceFile ]; then
     __versionSourceLine="static char sccsid[] __attribute__((used)) = \"@(#)No version information produced\";"
-- 
2.24.0

