From 82fe479d9344448b3b80881b2aed327713224ace Mon Sep 17 00:00:00 2001
From: s0dyy <maxime.sorin@clever-cloud.com>
Date: Thu, 11 Apr 2024 20:16:44 +0200
Subject: [PATCH] solr: hard coded pkg-config
Upstream: no

---
 config.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.m4 b/config.m4
index 11c7a19..01d7f2f 100644
--- a/config.m4
+++ b/config.m4
@@ -12,7 +12,7 @@ PHP_ARG_ENABLE(coverage, whether to enable code coverage,
 dnl Setting up the apache Solr extension
 if test "$PHP_SOLR" != "no"; then
 
-    AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+    AC_PATH_TOOL(PKG_CONFIG, pkg-config, no)
 
     AC_MSG_CHECKING(for libcurl)
     if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libcurl; then
-- 
2.44.0

