From 7d4802b2d469d139065a722c25d07751eaaa7402 Mon Sep 17 00:00:00 2001
From: s0dyy <maxime.sorin@clever-cloud.com>
Date: Thu, 5 Dec 2019 14:44:52 +0100
Subject: [PATCH] COREFX-Add-Exherbo-Group-RuntimeGroups

---
 pkg/Microsoft.NETCore.Platforms/runtimeGroups.props | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props b/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props
index eeb8130b..7ea4cda5 100644
--- a/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props
+++ b/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props
@@ -40,6 +40,11 @@
       <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="exherbo">
+      <Parent>linux</Parent>
+      <Architectures>x64</Architectures>
+    </RuntimeGroup>
+
     <RuntimeGroup Include="fedora">
       <Parent>linux</Parent>
       <Architectures>x64;arm64</Architectures>
-- 
2.24.0

