From 675c652ee103770d7d09fc7d514751d6aa479dca Mon Sep 17 00:00:00 2001
From: s0dyy <msorin@msorin.com>
Date: Tue, 22 Dec 2020 16:21:24 +0100
Subject: [PATCH] SOURCE-BUILD-add-RID-suffix

---
 .../PublishCoreSetupBinaries.cs                                  | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/PublishCoreSetupBinaries.cs b/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/PublishCoreSetupBinaries.cs
index 363fdee..f36f1e8 100644
--- a/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/PublishCoreSetupBinaries.cs
+++ b/tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/PublishCoreSetupBinaries.cs
@@ -41,6 +41,7 @@ namespace Microsoft.DotNet.Build.Tasks
                                               ".rhel.7", ".rhel.8", ".rhel.9",
                                               "-centos.7", "-centos.8", "-centos.9",
                                               ".centos.7", ".centos.8", ".centos.9",
+                                              ".exherbo", "-exherbo",
                                               ".fedora.30", "-fedora.30",
                                               ".fedora.31", "-fedora.31",
                                               ".fedora.32", "-fedora.32",
-- 
2.29.2

