From 64871e093159ad06d84fb2a84c7808a81800dfc4 Mon Sep 17 00:00:00 2001
From: Sage Weil <sage@inktank.com>
Date: Thu, 16 May 2013 09:42:11 -0700
Subject: [PATCH] mds: avoid assert after suicide()

Fixes: #5079
Signed-off-by: Sage Weil <sage@inktank.com>
---
 src/mds/MDCache.cc |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/mds/MDCache.cc b/src/mds/MDCache.cc
index 094aefd..cc661f2 100644
--- a/src/mds/MDCache.cc
+++ b/src/mds/MDCache.cc
@@ -5653,9 +5653,11 @@ void MDCache::_recovered(CInode *in, int r, uint64_t size, utime_t mtime)
 
   if (r != 0) {
     dout(0) << "recovery error! " << r << dendl;
-    if (r == -EBLACKLISTED)
+    if (r == -EBLACKLISTED) {
       mds->suicide();
-    assert(0);
+      return;
+    }
+    assert(0 == "unexpected error from osd during recovery");
   }
 
   file_recovering.erase(in);
-- 
1.7.2.5

