Author: Thomas Witt
Reason: Expected test output changed
Upstream-status: Pending

--- test/testlistgeo_out.dist.orig	2019-11-01 16:09:09.203054409 +0100
+++ test/testlistgeo_out.dist	2019-11-01 16:04:18.483057182 +0100
@@ -1697,11 +1697,11 @@
    Keyed_Information:
       GTModelTypeGeoKey (Short,1): ModelTypeProjected
       GTRasterTypeGeoKey (Short,1): RasterPixelIsArea
-      ProjectedCSTypeGeoKey (Short,1): Code-3035 (ETRS89 / LAEA Europe)
+      ProjectedCSTypeGeoKey (Short,1): Code-3035 (ETRS89-extended / LAEA Europe)
       End_Of_Keys.
    End_Of_Geotiff.
 
-PCS = 3035 (ETRS89 / LAEA Europe)
+PCS = 3035 (ETRS89-extended / LAEA Europe)
 Projection = 19986 (Europe Equal Area 2001)
 Projection Method: CT_LambertAzimEqualArea
    ProjCenterLatGeoKey: 52.000000 ( 52d 0' 0.00"N)
