Upstream: no
--
--- a/libtest/port.cc
+++ b/libtest/port.cc
@@ -150,7 +150,7 @@
         struct sockaddr_in sin;
         sin.sin_port= 0;
         sin.sin_addr.s_addr= 0;
-        sin.sin_addr.s_addr= INADDR_ANY;
+        sin.sin_addr.s_addr= inet_addr("127.0.0.1");
         sin.sin_family= AF_INET;
 
         int bind_ret;
--- a/tests/libmemcached-1.0/mem_functions.cc
+++ b/tests/libmemcached-1.0/mem_functions.cc
@@ -4493,7 +4493,7 @@
   in_port_t base_port= 11211;
   for (size_t x= 0; x < 17; x++)
   {
-    test_compare(MEMCACHED_SUCCESS, memcached_server_add(memc, "10.2.3.4", base_port +x));
+    test_compare(MEMCACHED_SUCCESS, memcached_server_add(memc, "127.0.0.1", base_port +x));
   }
   test_compare(6U, memcached_generate_hash(memc, test_literal_param("SZ6hu0SHweFmpwpc0w2R")));
   test_compare(1U, memcached_generate_hash(memc, test_literal_param("SQCK9eiCf53YxHWnYA.o")));
@@ -4601,7 +4601,7 @@
 {
   memcached_st *memc= memcached_create(NULL);
   test_true(memc);
-  test_compare(MEMCACHED_SUCCESS, memcached_server_add(memc, "10.2.3.4", 11211));
+  test_compare(MEMCACHED_SUCCESS, memcached_server_add(memc, "127.0.0.1", 11211));
 
   memcached_behavior_set(memc, MEMCACHED_BEHAVIOR_CONNECT_TIMEOUT, 1000);
   memcached_behavior_set(memc, MEMCACHED_BEHAVIOR_RETRY_TIMEOUT, 1000);
--- a/tests/libmemcached-1.0/parser.cc   2012-12-16 19:51:48 +0000
+++ b/tests/libmemcached-1.0/parser.cc   2013-01-07 13:12:52 +0000
@@ -737,11 +737,11 @@
 */
 test_return_t regression_bug_71231153_connect(memcached_st *)
 {
-  if (libmemcached_util_ping("10.0.2.252", 0, NULL)) // If for whatever reason someone has a host at this address, skip
+  if (libmemcached_util_ping("127.0.0.1", 0, NULL)) // If for whatever reason someone has a host at this address, skip
     return TEST_SKIPPED;
 
   { // Test the connect-timeout, on a bad host we should get MEMCACHED_CONNECTION_FAILURE
-    memcached_st *memc= memcached(test_literal_param("--SERVER=10.0.2.252 --CONNECT-TIMEOUT=0"));
+    memcached_st *memc= memcached(test_literal_param("--SERVER=127.0.0.1 --CONNECT-TIMEOUT=0"));
     test_true(memc);
     test_zero(memc->connect_timeout);
     test_compare(MEMCACHED_DEFAULT_TIMEOUT, memc->poll_timeout);
@@ -761,13 +761,13 @@
 
 test_return_t regression_bug_71231153_poll(memcached_st *)
 {
-  if (libmemcached_util_ping("10.0.2.252", 0, NULL)) // If for whatever reason someone has a host at this address, skip
+  if (libmemcached_util_ping("127.0.0.1", 0, NULL)) // If for whatever reason someone has a host at this address, skip
   {
     return TEST_SKIPPED;
   }
 
   { // Test the poll timeout, on a bad host we should get MEMCACHED_CONNECTION_FAILURE
-    memcached_st *memc= memcached(test_literal_param("--SERVER=10.0.2.252 --POLL-TIMEOUT=0"));
+    memcached_st *memc= memcached(test_literal_param("--SERVER=127.0.0.1 --POLL-TIMEOUT=0"));
     test_true(memc);
     test_compare(MEMCACHED_DEFAULT_CONNECT_TIMEOUT, memc->connect_timeout);
     test_zero(memc->poll_timeout);
