From bb8baf3851aa57cc3ae263f58dd65bacaec19fda Mon Sep 17 00:00:00 2001
From: Lucas Ransan <lucas.ransan@clever-cloud.com>
Date: Wed, 5 Jul 2023 17:24:05 +0200
Subject: [PATCH] make mdbook linkcheck optional

---
 doc/manual/book.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/manual/book.toml b/doc/manual/book.toml
index 73fb7e75e..ae8044760 100644
--- a/doc/manual/book.toml
+++ b/doc/manual/book.toml
@@ -12,6 +12,7 @@ renderers = ["html"]
 command = "jq --from-file doc/manual/anchors.jq"
 
 [output.linkcheck]
+optional = true
 # no Internet during the build (in the sandbox)
 follow-web-links = false
 
-- 
2.41.0

