From 2b282bdbdb3455849209a0c1bab4030a428d79f7 Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Wed, 13 Feb 2013 09:39:39 +0100
Subject: [PATCH] haproxy: Don't install .in file

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
---
 packages/net/haproxy/haproxy.exlib | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/net/haproxy/haproxy.exlib b/packages/net/haproxy/haproxy.exlib
index 9a7d382..dab4794 100644
--- a/packages/net/haproxy/haproxy.exlib
+++ b/packages/net/haproxy/haproxy.exlib
@@ -3,7 +3,7 @@
 
 if ever at_least "1.5-pre1"; then
     # TODO: file instead of dir
-    require systemd-service [ systemd_files=[ contrib/systemd ] ]
+    require systemd-service [ systemd_files=[ contrib/systemd/${PN}.service ] ]
 else
     require systemd-service
 fi
-- 
1.8.1.2

