Upstream: no
--
--- a/glusterfsd/src/Makefile.am
+++ b/glusterfsd/src/Makefile.am
@@ -23,7 +23,6 @@
 	rm -f $(DESTDIR)$(sbindir)/glusterd

 install-data-local:
-	$(INSTALL) -d -m 755 $(DESTDIR)$(localstatedir)/run
 	$(INSTALL) -d -m 755 $(DESTDIR)$(localstatedir)/log/glusterfs
 	rm -f $(DESTDIR)$(sbindir)/glusterfs
 	rm -f $(DESTDIR)$(sbindir)/glusterd
--- a/glusterfsd/src/Makefile.in
+++ b/glusterfsd/src/Makefile.in
@@ -533,7 +533,6 @@
 	rm -f $(DESTDIR)$(sbindir)/glusterd

 install-data-local:
-	$(INSTALL) -d -m 755 $(DESTDIR)$(localstatedir)/run
 	$(INSTALL) -d -m 755 $(DESTDIR)$(localstatedir)/log/glusterfs
 	rm -f $(DESTDIR)$(sbindir)/glusterfs
 	rm -f $(DESTDIR)$(sbindir)/glusterd
