From 51dc2061f69c1d4f8774d80cca730b7b1224b3dc Mon Sep 17 00:00:00 2001
From: s0dyy <maxime.sorin@clever-cloud.com>
Date: Sat, 29 Feb 2020 22:26:00 +0100
Subject: [PATCH] Systemd-Fix-Legacy-Path-PID

---
 Installation/systemd/arangodb3.service.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Installation/systemd/arangodb3.service.in b/Installation/systemd/arangodb3.service.in
index 7a095956d1..e573c37af8 100644
--- a/Installation/systemd/arangodb3.service.in
+++ b/Installation/systemd/arangodb3.service.in
@@ -25,14 +25,14 @@ LimitNOFILE=131072
 LimitNPROC=131072
 TasksMax=131072
 
-PIDFile=/var/run/arangodb3/arangod.pid
+PIDFile=/run/arangodb3/arangod.pid
 Environment=GLIBCXX_FORCE_NEW=1
 
 # Protect users from making their installation unusable by
 # starting arangod with wrong permissions (e.g. as root).
 # This will reset the permissions to the working default.
 ExecStartPre=/usr/bin/install -g arangodb -o arangodb -d /var/tmp/arangodb3
-ExecStartPre=/usr/bin/install -g arangodb -o arangodb -d /var/run/arangodb3
+ExecStartPre=/usr/bin/install -g arangodb -o arangodb -d /run/arangodb3
 ExecStartPre=@CHOWN_EXECUTABLE@ -R arangodb:arangodb /var/log/arangodb3
 ExecStartPre=@CHMOD_EXECUTABLE@ 700 /var/log/arangodb3
 ExecStartPre=@CHOWN_EXECUTABLE@ -R arangodb:arangodb /var/lib/arangodb3
@@ -40,7 +40,7 @@ ExecStartPre=@CHMOD_EXECUTABLE@ 700 /var/lib/arangodb3
 ExecStartPre=@CHOWN_EXECUTABLE@ -R arangodb:arangodb /var/lib/arangodb3-apps
 ExecStartPre=@CHMOD_EXECUTABLE@ 700 /var/lib/arangodb3-apps
 
-ExecStart=/usr/sbin/arangod --uid arangodb --gid arangodb --pid-file /var/run/arangodb3/arangod.pid --temp.path /var/tmp/arangodb3 --log.foreground-tty true
+ExecStart=/usr/sbin/arangod --uid arangodb --gid arangodb --pid-file /run/arangodb3/arangod.pid --temp.path /var/tmp/arangodb3 --log.foreground-tty true
 
 TimeoutStopSec=3600
 TimeoutSec=3600
-- 
2.25.1

